src/content/operations/cleanup/_index.en.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Uninstalling Submariner"
+date: 2020-12-23T21:25:11+01:00
+weight: 20
+---
+
+To properly uninstall Submariner from a cluster, follow the steps below.
+
+1. Delete Submariner related namespaces:
+
+   On data cluster,
+
+   ```bash
+   kubectl delete namespace submariner-operator
+   ```
+
+   On broker cluster,
+
+   ```bash
+   kubectl delete namespace submariner-k8s-broker
+    ```
+
+2. Deleting submariner CRDs in the cluster
+
+   ```bash
+   for CRD in `kubectl get crds | grep -iE 'submariner|multicluster.x-k8s.io'| awk '{print $1}'`; do kubectl delete crd $CRD; done
+   ```
+
+3. Delete ClusterRoles and ClusterRoleBindings
+
+   ```bash
+   roles="submariner-operator submariner-operator-globalnet submariner-lighthouse"
+   kubectl delete clusterrole,clusterrolebinding $roles
```

Comment:
```suggestion
   kubectl delete clusterrole,clusterrolebinding $roles --ignore-not-found
```
Just in case one of the objects does not exist try to avoid unnecessary errors.